packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -723,6 +818,63 @@ onMounted(async () => {
 	initialize();
 });
 
+const preparedInterventionsCharts = computed(() => {
+	const preForecastRunId = knobs.value.preForecastRunId;
+	const postForecastRunId = knobs.value.postForecastRunId;
+	if (!postForecastRunId || !preForecastRunId) return [];
+	const preResult = runResults.value[preForecastRunId];
+	const preResultSummary = runResultsSummary.value[preForecastRunId];
+	const postResult = runResults.value[postForecastRunId];
+	const postResultSummary = runResultsSummary.value[postForecastRunId];
+
+	return knobs.value.selectedInterventionVariables.map((variable) =>
+		createOptimizeForecastChart(
+			preResult,
+			preResultSummary,
+			postResult,
+			postResultSummary,
+			preProcessedInterventionsData.value[variable],
+			{
+				width: chartSize.value.width,
+				height: chartSize.value.height,
+				variables: [pyciemssMap[variable]],
+				statisticalVariables: [`${pyciemssMap[variable]}_mean`],
+				legend: false,
+				groupField: 'sample_id',
+				timeField: 'timepoint_id',
+				xAxisTitle: 'Time',
+				yAxisTitle: variable,
+				title: variable
+			}
+		)
+	);
+});
+
+const preparedCharts = computed(() => {
+	const preForecastRunId = knobs.value.preForecastRunId;
+	const postForecastRunId = knobs.value.postForecastRunId;
+	if (!postForecastRunId || !preForecastRunId) return [];
+	const preResult = runResults.value[preForecastRunId];
+	const preResultSummary = runResultsSummary.value[preForecastRunId];
+	const postResult = runResults.value[postForecastRunId];
+	const postResultSummary = runResultsSummary.value[postForecastRunId];
+
+	return knobs.value.selectedSimulationVariables.map((variable) =>
+		createOptimizeForecastChart(preResult, preResultSummary, postResult, postResultSummary, [], {
+			width: chartSize.value.width,
+			height: chartSize.value.height,
+			variables: [pyciemssMap[variable]],
+			statisticalVariables: [`${pyciemssMap[variable]}_mean`],
+			legend: false,
+			groupField: 'sample_id',
+			timeField: 'timepoint_id',
+			xAxisTitle: 'Time',
+			yAxisTitle: variable,
+			title: variable
+		})
+	);
+});
```

Comment:
This is the same code as in the node, why it is not in the operation.ts file?

Reply:
they are slightly different, the node view does have label or legends for a more condensed view while the drilldown view has more detail